src/js/disability-benefits/526EZ/components/ReviewCollapsiblePage.jsx

```diff
@@ -0,0 +1,157 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Scroll from 'react-scroll';
+import _ from 'lodash/fp';
+import classNames from 'classnames';
+
+import { focusElement } from '../../../common/utils/helpers';
+import SchemaForm from '../../../common/schemaform/components/SchemaForm';
+import ProgressButton from '../../../common/components/form-elements/ProgressButton';
+
+const Element = Scroll.Element;
+const scroller = Scroll.scroller;
+export default class ReviewCollapsiblePage extends React.Component {
```

Comment:
I think we should name this something else now. It's not collapsible

Reply:
Great point, will rename this `verifiedReviewPage`, and rename `verifiedReviewPage`, `verifiedReviewContainer`.